spec/beatable_ai_player_spec.rb

```diff
@@ -0,0 +1,20 @@
+require "beatable_ai_player"
+
+RSpec.describe BeatableAIPlayer do
+  let(:display_spy) { instance_spy(ConsoleUI) }
+  let(:beatable_player) { BeatableAIPlayer.new(Mark::X, display_spy) }
+
+  it "knows who it's opponent is" do
+    expect(beatable_player.get_opponent_mark).to eq(Mark::O)
+  end
+
+  it "gets next position from console" do
+    expect(beatable_player.get_next_move).to be_between(1, 9).inclusive 
```

Comment:
Would it be better to pass in the random generator to the player, so that you can plant what number will be returned in test? Maybe it wouldn't give you anything more though..